packages/rocketchat-videobridge/client/public/external_api.js

```diff
@@ -0,0 +1,377 @@
+ */
+var JitsiMeetExternalAPI = (function()
+{
+    /**
+     * The minimum width for the Jitsi Meet frame
+     * @type {number}
+     */
+    var MIN_WIDTH = 790;
+
+    /**
+     * The minimum height for the Jitsi Meet frame
+     * @type {number}
+     */
+    var MIN_HEIGHT = 300;
+
+    /**
+     * Constructs new API instance. Creates iframe element that loads
+     * Jitsi Meet.
+     * @param domain the domain name of the server that hosts the conference
+     * @param room_name the name of the room to join
+     * @param width width of the iframe
+     * @param height height of the iframe
+     * @param parent_node the node that will contain the iframe
+     * @param filmStripOnly if the value is true only the small videos will be
+     * visible.
+     * @constructor
+     */
+    function JitsiMeetExternalAPI(domain, room_name, width, height, parentNode,
+        configOverwrite, interfaceConfigOverwrite) {
+        if (!width || width < MIN_WIDTH)
+            width = MIN_WIDTH;
+        if (!height || height < MIN_HEIGHT)
+            height = MIN_HEIGHT;
+
+        this.parentNode = null;
+        if (parentNode) {
+            this.parentNode = parentNode;
+        } else {
+            var scriptTag = document.scripts[document.scripts.length - 1];
+            this.parentNode = scriptTag.parentNode;
+        }
+
+        this.iframeHolder =
+            this.parentNode.appendChild(document.createElement("div"));
+        this.iframeHolder.id = "jitsiConference" + JitsiMeetExternalAPI.id;
+        if(width)
+            this.iframeHolder.style.width = width + "px";
+        if(height)
+            this.iframeHolder.style.height = height + "px";
+        this.frameName = "jitsiConferenceFrame" + JitsiMeetExternalAPI.id;
+        this.url = "https://" + domain + "/";
+        if(room_name)
+            this.url += room_name;
+        this.url += "#external=true";
+
+        var key;
+        if (configOverwrite) {
+            for (key in configOverwrite) {
+                if (!configOverwrite.hasOwnProperty(key) ||
+                    typeof key !== 'string')
+                    continue;
+                this.url += "&config." + key + "=" + configOverwrite[key];
+            }
+        }
+
+        if (interfaceConfigOverwrite) {
+            for (key in interfaceConfigOverwrite) {
+                if (!interfaceConfigOverwrite.hasOwnProperty(key) ||
+                    typeof key !== 'string')
+                    continue;
+                this.url += "&interfaceConfig." + key + "=" +
+                    interfaceConfigOverwrite[key];
+            }
+        }
+
+        JitsiMeetExternalAPI.id++;
+
+        this.frame = document.createElement("iframe");
+        this.frame.src = this.url;
+        this.frame.name = this.frameName;
+        this.frame.id = this.frameName;
+        this.frame.width = "100%";
+        this.frame.height = "100%";
+        this.frame.setAttribute("allowFullScreen","true");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2737452894/issues/source?bid=3199634&fileBranchId=3342079#l86)